RELEASENOTES/v2.0.0.alpha-3.MD

```diff
@@ -0,0 +1,51 @@
+# Notary v2 Alpha 3 Release
+
+## Overview
+Notary v2 working group is pleased to announce the third release, Alpha 3. Refer release definitions [here](https://github.com/notaryproject/notation/blob/main/RELEASE_MANAGEMENT.md). This release includes updates to the [notaryproject/notaryproject](https://github.com/notaryproject/notaryproject) repo to provide the community with updated specifications and an updated implementation of Notation client for signing and verifying artifacts.
+
+- [Notation Library 1](https://github.com/notaryproject/notation-core-go) ***[(notation-go-v1.0.0-alpha.3)](https://github.com/notaryproject/notation-core-go/releases/tag/v0.1.0-alpha.3)***   (Added in this release)
```

Comment:
notation-go-v1.0.0-alpha.3 --> notation-core-go-v0.1.0-alpha.3

Reply:
fixed, and also added "NotaryProject" release for the specs. Also took out the mention of "directory based structure" and "JSON document based Trust policy" from this alpha-3 system release notes.

cc: @dtzar , @vaninrao10 , @priteshbandi , @yizha1